README.md

```diff
@@ -11,49 +11,171 @@
 
 
 
-Library to propose code changes
+## Description
+Code-suggester automates the steps involved in making code changes or code suggestions to your [GitHub](https://github.com/) repository! Code-suggester
+1. can be imported as a [library](#Core-Library), or
+2. used as a [CLI](#CLI) tool
 
 
+## Core Library
 
+### Installation
 
-* [github.com/googleapis/code-suggester](https://github.com/googleapis/code-suggester)
+```bash
+npm i code-suggester
+```
+
+### Example
+
+```
+import { makePr } from "code-suggester";
```

Comment:
This makes one of two assumptions :) Either a.) the user is using typescript, or b.) they are using [es-modules](https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/).  We almost always assume users are using neither of those, because the majority of folks are using JavaScript and CommonJS.  This may not matter if we're only serving TypeScript users, but FYI!